docs/multi-homing.md

```diff
@@ -0,0 +1,117 @@
+# Multi-homing
+A K8s pod with more than one network interface is said to be multi-homed. The
+[Network Plumbing Working Group](https://github.com/k8snetworkplumbingwg/multi-net-spec)
+has put forward a [standard](https://github.com/k8snetworkplumbingwg/multi-net-spec)
+describing how to specify the configurations for additional network interfaces.
+
+There are several delegating plugins or meta-plugins (Multus, Genie) implementing this standard.
+
+After a pod is scheduled on a particular Kubernetes node, kubelet will invoke
+the delegating plugin to prepare the pod for networking. This meta-plugin will
+then invoke the CNI responsible for setting up the pod's default cluster
+network, and afterwards it iterates the list of additional attachments on the
+pod (yes, these plugins **must** be able to access the Kubernetes API...),
```

Comment:
```suggestion
pod,
```

This additional note does not seem necessary. The "yes" is answering a question I never had

Reply:
Done.